Annex60/Fluid/Movers/Data/SpeedControlled_Nrpm.mo

```diff
@@ -5,12 +5,21 @@ record SpeedControlled_Nrpm "Generic data record for FlowMachine_Nrpm"
   parameter Modelica.SIunits.Conversions.NonSIunits.AngularVelocity_rpm N_nominal = 1500
     "Nominal rotational speed for flow characteristic";
 
+  parameter Real[:] speeds(each final unit="1/min") = {0}
```

Comment:
For 1): I copied the implementation from `Annex60.Fluid.Movers.SpeedControlled_Nrpm` so I'll change it there too.
For 2): the variable name `N` does not really reflect that the variable is an array. Also to be consistent we'd have to change `Annex60.Fluid.Movers.SpeedControlled_Nrpm`, which is non-backwards compatible. I also think it is easier to interpret the meaning of the parameter when using `speeds`. So I would propose to stick with the name `speeds`?